core-bundle/contao/languages/en/tl_page.xlf

```diff
@@ -320,6 +320,12 @@
       <trans-unit id="tl_page.hide.1">
         <source>Hide the page in the navigation menu.</source>
       </trans-unit>
+      <trans-unit id="tl_page.guests.0">
+        <source>Hide in navigation if logged in</source>
+      </trans-unit>
+      <trans-unit id="tl_page.guests.1">
+        <source>Hide the page in the menu if a member is logged in.</source>
```

Comment:
```suggestion
        <source>Hide the page in navigation modules if a member is logged in.</source>
```

Reply:
This might need to be changed with navigation content-elements as well, no? 😁

Reply:
The description of `tl_page.hide` says

> Hide the page in the navigation menu.

So may be for consistency it should be

> Hide the page in the navigation menu if a member is logged in.